### DISCUSSION TITLE
Who is the world’s longest-serving current president?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#785] [Galaxy Trivia] [answered]
**Bonus:** What is the tree on the national flag?

**Comment by SANKARAMDAS** [comment, marked as answer]
The longest-serving and longest current serving president ever is Teodoro Obiang Nguema Mbasogo, who is currently the President of the Republic of Equatorial Guinea, who seized power in a coup in 1979.

A silk-cotton tree, also known as the god tree,

**Comment by peterbaccaro** [comment]
The longest-serving and longest current serving male president ever is **Teodoro Obiang Nguema Mbasogo**, who is currently the President of the Republic of Equatorial Guinea, who seized power in a coup in 1979.

The coat of arms of Equatorial Guinea contains the **silk-cotton tree**, also known as the **god tree**, under which, it is said, the first treaty was signed between Spain and a local ruler.
